### DISCUSSION TITLE
Zenodo: add graphical report + link to Bionomia profile

### DISCUSSION THREAD

**Discussion by abubelinha** [bionomia/bionomia#347] [Ideas] [answered]
I've been taking a look at Zenodo integration (great idea BTW).

It just includes text files of claimed specimens. 
I wouldn't say it's ugly but ...

I think it would be nice to see a small PDF report with some of the summary visual info you can see in the web site.
(not just claims but also the collaborative help given by that user): 

- specimens collected (simple world map / taxonomic groups)
- specimens identified (simple world map / taxonomic groups)
- help provided in Bionomia attribution works (some graphical account of people helped and number of specimens attributed)
  i.e. this bar graph https://bionomia.net/profile/helped
- ... whatever

Thanks 
@abubelinha

**Comment by abubelinha** [comment]
Also, current HTML description says:

> Natural history specimen data collected and/or identified by _User Name_, https://orcid.org/xxxx-xxxx-xxxx-xxxx. Claims or attributions were made on Bionomia, [https://bionomia.net](http://bionomia.net/) using specimen data from the Global Biodiversity Information Facility, https://gbif.org/.

I think it would be interesting to include a link back to Bionomia user's profile.

**Reply by dshorthouse** [reply]
Easy to implement to be sure, but you need not make your profile public in Bionomia yet still push to Zenodo. There are a few examples of that.

**Reply by abubelinha** [reply]
It is also interesting to include in this report:

### helped/deposited datasets (not only helped people).

- full list of **datasets** with number of data attributed/recorded/identified by me
- list of **countries**: number of datasets and attributed/recorded/identified data, grouped by dataset country (different of record location country!!)

I'd reverse sort both lists by sum A+R+D 

### Final citation of people who helped me

(Kind of bibliography ... as scribes also helped to generate this report)

**Reply by abubelinha** [reply]
> _you need **not**_ make your profile public in Bionomia yet still push to Zenodo. 

You main "_you need_", don't you?

With "push to Zenodo" you mean the report is just created once but not regularly updated if user don't do that "push" again?
Is this documented somewhere?

**Reply by dshorthouse** [reply, marked as answer]
The push to Zenodo is automated & re-executed on a schedule, dependent on activity: https://bionomia.net/how-it-works

**Reply by abubelinha** [reply]
>Easy to implement to be sure ... make your profile public in Bionomia yet still push to Zenodo
>The push to Zenodo is automated & re-executed on a schedule, dependent on activity: https://bionomia.net/how-it-works

Thanks. I already did that and I looked at https://bionomia.net/how-it-works
> every week on your behalf, a new version of your specimen data is pushed to the archive when you make new claims. 

Does this happen on a certain weekday for all users, or is this each 7 days accounting from the original Zenodo push? 
(i.e. on a different weekday for each user)
I long for those nice pdf reports to show up 😅

**Comment by dshorthouse** [comment]
> Does this happen on a certain weekday for all users, or is this each 7 days accounting from the original Zenodo push? (i.e. on a different weekday for each user) I long for those nice pdf reports to show up 😅

Every 23:00 UTC on Mondays, they're queued up for all users that have had a new attribution/claim made in the previous 7 days, both ORCID- (using cached OAuth token) or wikidata-based accounts. Zenodo also checks upload hashes against the current version and so may kick out a new upload attempt.
